doc/requirements.txt

```diff
@@ -0,0 +1,11 @@
+Sphinx >= 3.5.3
+graphviz
+mkdocs
+sphinx-rtd-theme
+sphinxcontrib-contentui
+sphinxcontrib-fancybox
+
+appdirs >= 1.4.0
```

Comment:
Are these bits all needed?  And how careful do we need to be about keeping them in sync with setup.py?

Reply:
Was this resolved?

Reply:
Bump.